src/06-sort.js

```diff
@@ -33,7 +44,16 @@ function sortByArtistNameDescending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortBySongTitleAscending(songs) {}
+function sortBySongTitleAscending(songs) {
+  return songs.sort((a,b) => {
+    switch (false) {
```

Comment:
Make sure to include the possibility that a === b just to leave no room for error